homeassistant/components/device_tracker/__init__.py

```diff
@@ -78,7 +80,10 @@ def setup(hass, config):
     seconds = util.convert(config[DOMAIN].get(CONF_SECONDS), int,
                            DEFAULT_CONF_SECONDS)
 
-    tracker = DeviceTracker(hass, device_scanner, seconds)
+    track_new_devices = config[DOMAIN].get(TRACK_NEW_DEVICES) or False
```

Comment:
The `get` method takes in a second argument that is the return value if the key is not found.